includes/results.inc

```diff
@@ -1021,7 +1021,11 @@ class IslandoraSolrFacets {
       $link['count'] = $count;
       $link['link_plus'] = '<a' . drupal_attributes($attributes['plus']['attr']) . '>+</a>';
       $link['link_minus'] = '<a' . drupal_attributes($attributes['minus']['attr']) . '>-</a>';
-      $buckets[] = $link;
+      $buckets[$bucket] = $link;
+    }
+
+    if (isset($this->settings['solr_field_settings']['sort_by']) && $this->settings['solr_field_settings']['sort_by'] == 'index') {
+      ksort($buckets);
```

Comment:
@whikloj 

Just trying to avoid confusion... The [JIRA ticket](https://jira.duraspace.org/browse/ISLANDORA-1337) indicates:

> This would implement facet.sort ...

... which is not happening here... Should this be a distinct ticket?

Also, I feel that there may be mixups between the two... Could expand on the description beside the option, to be more explicit in the difference.

... Dealing with the transformations... I'm not sure there's any clean way to do it generally... Almost feeling like it may be more reliably done with something concatenating things together into a purpose-built field in the index... something like `the text to display/the:pid`), and having something strip off the `/the:pid` (some arbitrary separator could be used instead of a forward-slash... just using it here as an example) bit when displaying... Dunno...